src/lib/media/manip.ts

```diff
@@ -182,12 +202,28 @@ async function moveToPermanentPath(path: string, ext = ''): Promise<string> {
   https://github.com/ivpusic/react-native-image-crop-picker/issues/1199
   */
   const filename = uuid.v4()
+  const destinationPath = joinPath(cacheDirectory ?? '', `${filename}${ext}`)
+
+  await copyAsync({
+    from: normalizePath(path),
+    to: destinationPath,
+  })
+
+  // This is just to try and clean up whenever we can. We won't always be able to, so in cases where we can't
+  // we just catch the error.
+  // Paths that are image caches shouldn't be removed. com.hackemist.SDImageCache is used by SDWebImage and
+  // image_manager_disk_cache is used by Glide
+  if (
+    !path.includes('com.hackemist.SDImageCache') &&
+    !path.includes('image_manager_disk_cache')
+  ) {
+    try {
+      deleteAsync(path)
+    } catch (e) {
+      // No need to handle
+    }
```

Comment:
This is new logic, but it has no affect on the client. It only removes old files.